devdocs/process.md

```diff
@@ -82,8 +82,10 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
      doesn't always result in a good layout.
         
    * Ensure _dev green_ (i.e., all tests are passing on dev server).
+
+    >If any of the browsertests fail, use [GodMode](https://docs.google.com/) to fix them. Run the test suite again without GodMode and ensure _dev green_
```

Comment:
That's still not a 100% guarantee because the developer can still change it manually. But if we go by the "something is better than nothing" principle, then sure it's one way to go.